week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -33,6 +34,8 @@ console.log(y)
 ```
 
 What will be the output of this code. Explain your answer in 50 words or less.
+After the function is called the console inside the function will display 10 then te last console will display undefined as y can not be accessed outside its block.
```

Comment:
- the function has been executed inside the console.log.
- f1 does not have return statement so it will log either undefined or log the function statement .
- it will logs x anyway  while it's been executed 
- y undefined.